pkg/securitypolicy/securitypolicy.go

```diff
@@ -104,6 +111,9 @@ type ExecProcessConfig struct {
 	Signals []syscall.Signal `json:"signals" toml:"signals"`
 }
 
+const apiSVN = "0.10.0"
```

Comment:
Is there a way to put these in a separate file (json or other format of config file) and then consume the file? It will make it easier for the ACC-CLI repo to consume and keep our versioning up to date with yours.

Reply:
I've moved the SVN values to separate files named `svn_api` and `svn_framework`

Reply:
Works for me. thanks!